README.md

```diff
@@ -23,6 +35,38 @@ The Azure Devops pipeline is configured to install prerequisites, run the tests,
 
 When a test fails in the pipeline, Playwright is configured to retry it and record a trace - this trace can be analysed to see the entire flow of the test including the full dom before and after each Playwright action. See [Playwright Trace docs](https://playwright.dev/docs/trace-viewer) for information on how to view the trace.
 
-## Common problems
+## Page Object Model pattern
+
+This exemplar showcases a Page Object Model pattern. Good use of this pattern will give maximum reuse of components between different tests whilst making the tests simple to read and understand.
+
+### Structure
+
+Within the tests folder there is a sub folder for each type of test and a sub folder for the page object model. The page object model is shared between the tests.
+
+The `page-object-model` folder contains a file for each web page - within each file there is a `Page` class that is responsible for interacting with the page and a `PageAssertions` class which is responsible for asserting behaviour of a page.
+
+The `BasePage` contains properties and behaviour common to all the pages. 
+
+The `PageUtilities` class is accessible through the `BasePage` and is for methods that could be used on any page but are independent
+
+## Do/do nots
+
+### Do
+
+* Use [`locator`](https://playwright.dev/docs/locators) to select elements
+* Follow [best practice](https://playwright.dev/docs/selectors#best-practices) by using user-facing attributes such as text or aria to select elements
+* Use the page object model to retrieve information from the screen, drive the application and assert that screen elements are correct
+* Create `locator` properties at the top of the relevant `Page` class and populate them in the constructor. Use these properties to access the screen
+* Use [locator assertions](https://playwright.dev/docs/test-assertions) to verify screen content
+
+### Do not
+* **Never** use `$` or `$$` to select elements - use [`locator`](https://playwright.dev/docs/locators) to select elements
```

Comment:
Why not?  Can you provide a reason, or link to justification.  It would be useful to have the 'why' ready when we are code reviewing!  Also for the line below.

Reply:
I'm cautious about overloading these bullets with detail as I want them to be easy to scan through, also the intellisense and Playwright docs are both very clear and prominent about not using ElementHandle (anything starting with $) so this really shouldn't require further explanation.

If you're interested though...
> The difference between the [Locator](https://playwright.dev/docs/api/class-locator) and [ElementHandle](https://playwright.dev/docs/api/class-elementhandle) is that the [ElementHandle](https://playwright.dev/docs/api/class-elementhandle) points to a particular element, while [Locator](https://playwright.dev/docs/api/class-locator) captures the logic of how to retrieve an element.
This means that upon a partial/full page refresh (not something you can control) an element captured by ElementHandle becomes lost and the test will break. This is the pattern that Selenium uses and also one of the most common reasons that Selenium tests break. By using this in Playwright tests we compromise stability for no reason.